main.cpp

```diff
@@ -42,7 +42,7 @@ int main()
 
   string configuration_board = "forward_camera";
   int number_stereoboard = 1;
-  int number_take = 16;	// 3
+  int number_take = 3;	// 3
   float time_inc = 1/7.; // 1/8
```

Comment:
you are changing the default take but not the time_inc. Take 16 is 1/7 and take 3 is 1/8 (as shown on the original file)